src/main/resources/project/ec_setup.pl

```diff
@@ -510,6 +511,15 @@
                                         stepName      => 'Get Operation Status'
                                      }
                                     );
+
+            $batch->attachCredential(
+                                     "\$[/plugins/$pluginName/project]",
+                                     $cred,
+                                     {
+                                        procedureName => 'Create VM',
+                                        stepName      => 'Create VM'
```

Comment:
The procedure name referenced here does not match the name above 'Create Virtual Machine'